exporter/opensearchexporter/logs_exporter.go

```diff
@@ -0,0 +1,104 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// Package opensearchexporter contains an opentelemetry-collector exporter
+// for OpenSearch.
+package opensearchexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opensearchexporter"
+
+import (
+	"context"
+	"fmt"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.uber.org/multierr"
+	"go.uber.org/zap"
+)
+
+type opensearchLogsExporter struct {
+	logger *zap.Logger
+
+	index       string
+	maxAttempts int
+
+	client      *osClientCurrent
+	bulkIndexer osBulkIndexerCurrent
+	model       mappingModel
+}
+
+var retryOnStatus = []int{500, 502, 503, 504, 429}
+
+const createAction = "create"
```

Comment:
Are there project standards to use all capitalized variable names for constants?